apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -622,24 +654,29 @@ function initializeBlocklyWrapper(blocklyInstance) {
     blocksInWorkspace
       .filter(block => block.getParent() === null)
       .forEach(block => {
-        const svgTransformList = block.svgGroup_.transform.baseVal;
+        const svgTransformList = block.getSvgRoot().transform.baseVal;
         const svgTransform = svgTransformList.getItem(0);
         const svgTranslationX = svgTransform.matrix.e;
         svgTransform.setTranslate(svgTranslationX, 0);
       });
 
     // Shrink SVG to size of the block
-    const bbox = svg.getBBox();
-    svg.setAttribute('height', bbox.height + bbox.y);
-    svg.setAttribute('width', bbox.width + bbox.x);
+    const bbox = (svg as SVGGraphicsElement).getBBox();
+    svg.setAttribute('height', `${bbox.height + bbox.y}`);
```

Comment:
`setAttribute` expects a string, not a number.

Reply:
Was this just implicitly converting to a string before then? Seems better to be explicit if we can.